genon/preprocessor/facade/attachment_processor_guardrail.py

```
@@ -0,0 +1,1626 @@
+    async def __call__(self, request: Request, file_path: str, **kwargs: dict):
+        document: DoclingDocument = self.load_documents(file_path, **kwargs)
+        artifacts_dir, reference_path = self.get_paths(file_path)
+        document = document._with_pictures_refs(image_dir=artifacts_dir, reference_path=reference_path)
+
+        chunks: list[DocChunk] = self.split_documents(document, **kwargs)
+
+        vectors = []
+        if len(chunks) >= 1:
+            vectors: list[dict] = await self.compose_vectors(document, chunks, file_path, request, **kwargs)
+        else:
+            raise GenosServiceException(1, f"chunk length is 0")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)